README.md

```diff
@@ -0,0 +1,87 @@
+# multiplex-reservation
+
+## Table of contents
+* [Informacje Ogólne](#informacje-ogólne)
+* [Komunikacja z aplikacją](#komunikacja-z-aplikacją)
+* [Warstwa Service](#warstwa-service)
+* [Generowanie Kina](#generowanie-kina)
+* [Odnośnie wymagań](#odnośnie-wymagań)
+
+## Informacje Ogólne
+
+Aplikacja webowa napisana w języku Java z użyciem frameworka Spring umozliwuająca rezerwację miejsc w kinie typu multiplex.
+Aplikacja nie posiada zaimplementowanego frontendu. Komunikowałem się z nią i testowałem za pomocą aplikacji Postman.
+Aplikacja korzysta z JPA oraz bazy danych H2 pod URL *"http://localhost:8080/console"* z adresem URL JDBC *"jdbc:h2:file:./bazaDanych"*.
+
+Baza danych zbudowana jest za pomocą 6 encji: User, Reservation, Screening, Movie, CinemaRoom, Seat.
+Każda z nich posiada swoje prywatne atrybuty oraz jest połączona z odpowiednimi innymi encjami.
+Każda encja korzysta z Repozytorium, które implementuje *PagingAndSortingRepository*.
+
+## Komunikacja z aplikacją
+
+Apliakcja obsługuje REST API i korzysta z dwóch komunikatów *GET* i *POST* w wielu wariantach:
+
+* Wypisanie pokazów filmowych konkretnego dnia w konkretnym przedziale czasowym za pomocą scieżki: 
+*/multiplex/reservation/screening/interval*
+
+Podać należy 3 paramentry w następującej kolejności: dzień, godzina *Od*, godzina *Do*.  
```

Comment:
1. Czemu ta kolejność jest ważna?
2. Czy masz pomysł jak te parametry można by ograć inaczej, np tak aby obejrzeć repertuar na cały weekend?
3. Czemu nie możemy podać np 17:30?

Reply:
Ad 1: Gdyby ktoś podał w innej kolejności to program by to inaczej zinterpretował i najprawdopodobniej nie wyświetlił tego czego użytkownik oczekiwał.
Ad 2: Jest zaimplementowana funkcja RESTowa, która wyświetla wszystkie repertuary od zadanej godziny. Nie była ona wymagana więc jej nie opisywałem, ale istnieje i działa. Mogłaby dostać jako argumenty np aktualną godzinę i wyświetlić wszystko co będzie od teraz. Miałem pomysł aby użyć zmiennych LocalDate i LocalTime, ale wymagałoby to zamiany dużej części kodu i już nie chciałem eksperymentować przed deadlinem.
Ad 3: Tak jak wyżej, miałem już pomysł jak zaimplementować "minuty", których aktualnie ten program nie obsługuje. Zbliżał się deadline i wolałem zaimplementować inne działania aplikacji niż minuty.